src/Features/Core/Portable/Shared/TestHooks/FeatureAttribute_Names.cs

```diff
@@ -4,6 +4,7 @@ namespace Microsoft.CodeAnalysis.Shared.TestHooks
 {
     internal partial class FeatureAttribute
     {
+        public const string ABTest = nameof(ABTest);
```

Comment:
Forgot to remove when I moved it back to single-threaded, it was for scheduling.